Quadriga/src/main/java/edu/asu/spring/quadriga/accesschecks/impl/ProjectSecurityChecker.java

```diff
@@ -190,34 +189,14 @@ public boolean checkProjectAccess(String userName, String projectId)
         chkAccess = this.isProjectOwner(userName, projectId);
 
         if (!chkAccess) {
-            chkAccess = this.isUserCollaboratorOnProject(userName, projectId,
-                    RoleNames.ROLE_QUADRIGA_ADMIN);
+            chkAccess = this.isUserCollaboratorOnProject(userName, projectId, RoleNames.ROLE_QUADRIGA_ADMIN);
         }
         return chkAccess;
     }
 
-    /**
-     * This method checks if the user has the editor role to
-     * this Project
-     * 
-     * @param userName
-     * @return boolean - TRUE if the user is editor on the project; otherwise false
-     * @throws QuadrigaStorageException
-     * @author kiranbatna
-     */
-    @Override
-    @Transactional
-    public boolean isEditor(String userName,
-            String projectId) throws QuadrigaStorageException {
-        // check if the user is project editor
-        return accessManager
-                .isUserEditorOfProject(userName, projectId) > 0;
-    }
-
```

Comment:
"isEditor" method is removed completely and ''isCollaborator''  is used in place of this method.